pachyderm/templates/dash/tls-secret.yaml

```diff
@@ -1,15 +1,16 @@
+{{- /*
+SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
+SPDX-License-Identifier: Apache-2.0
+*/ -}}
 {{- /* Sanity check to ensure .create set if .crt or .key set  */ -}}
 {{- if and (not .Values.dash.tls.newSecret.create) .Values.dash.tls.newSecret.crt }}
   {{ fail "Must set dash.tls.newSecret.create to true when specifying dash.tls.newSecret.crt "}}
 {{ end -}}
 {{- if and (not .Values.dash.tls.newSecret.create) .Values.dash.tls.newSecret.key }}
   {{ fail "Must set dash.tls.newSecret.create to true when specifying dash.tls.newSecret.key "}}
 {{ end -}}
+{{- if .Values.dash.enabled -}}
 {{- if and .Values.dash.tls.enabled .Values.dash.tls.newSecret.create }}
```

Comment:
Could these two conditionals be combined? Looks like the inner one is the entirety of the outer’s body.

Reply:
Since this is a merge PR, I've made a follow on task to address https://github.com/pachyderm/helmchart/issues/128